merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinBindings.java

```diff
@@ -161,6 +161,8 @@ private void getSimulationResults(final Context ctx) {
       ctx.status(400).result(ResponseSerializers.serializeInvalidJsonException(ex).toString());
     } catch (final NoSuchPlanException ex) {
       ctx.status(404).result(ResponseSerializers.serializeNoSuchPlanException(ex).toString());
+    } catch (final MissionModelService.NoSuchMissionModelException ex) {
+      ctx.status(404).result(ResponseSerializers.serializeNoSuchMissionModelException(ex).toString());
```

Comment:
Thanks for adding this! Since this PR introduces `serializeNoSuchMissionModelException()` would you mind tacking this on to every `ctx.status(404)` related to a `MissionModelService.NoSuchMissionModelException` in this class?

This should keep things nice & consistent.

Reply:
done